_posts/en/newsletters/2025-01-17-newsletter.md

```diff
@@ -0,0 +1,148 @@
+  then allocate the funds to new DLCs.
+
+  Replies by Kulpreet Singh and Philipp Hoenisch linked to previous
+  research and development of this basic idea, including approaches that
+  allow the same pool of funds to be used for both offchain DLCs and
+  LN (see Newsletters [#174][news174 dlc-ln] and [#260][news260 dlc]).
+
+## Releases and release candidates
+
+_New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates._
+
+- [LDK v0.1.0-beta1][] is a release candidate of this library for
+  building LN-enabled wallets and applications.
+
+<!-- FIXME:harding to update Thursday -->
+
+## Notable code and documentation changes
+
+_Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Lightning BLIPs][blips repo], [Bitcoin Inquisition][bitcoin inquisition
+repo], and [BINANAs][binana repo]._
+
+- [Eclair #2936][] introduces a 12-block delay before marking a channel as
+  closed after its funding output has been spent to allow for the propagation of
+  a [splice][topic splicing] update (see Newsletter [#214][news214 splicing]).
+  Spent channels are temporarily tracked in a new `spentChannels` map, where
+  they are either removed after 12 blocks or updated as spliced channels. When a
+  splice occurs, the parent channel's short channel identifier (SCID), capacity,
+  and balance bounds are updated instead of creating a new channel.
+
+- [Rust Bitcoin #3792][] adds support for [BIP324][]’s [v2 P2P transport][topic
+  v2 P2P transport] to allow Bitcoin nodes to communicate with each other over
+  encrypted connections (see Newsletter [#306][news306 v2]). This is achieved by
+  adding a `V2NetworkMessage` method, which wraps a `NetworkMessage` method and
+  handles v2 encoding and decoding.
+
+- [BDK #1789][] updates the default transaction version from 1 to 2 to improve
+  wallet privacy since prior to this, BDK wallets were more identifiable due to
+  only 15% of the network using version 1. In addition, version 2 is required
+  for a future implementation of [BIP326][]’s nSequence-based [anti fee
+  sniping][topic fee sniping] mechanism for [taproot][topic taproot]
+  transactions.
+
+- [BIPs #1687][] merges [BIP375][] to specify sending [silent payments][topic
+  silent payments] using [PSBTs][topic psbt]. If there are multiple independent
+  signers, a DLEQ proof is required to allow all signers to prove to co-signers
+  that their signature is valid and doesn’t risk losing funds, without revealing
+  their private key (see Newsletter [#335][news335 dleq]).
```

Comment:
In addition to news335 could also reference pod327  https://bitcoinops.org/en/podcast/2024/11/05/#draft-bip-for-dleq-proofs where we had Andrew Toth on to explain